packages/calcite-components/src/components/input/input.tsx

```diff
@@ -343,6 +343,8 @@ export class Input
    * Specifies the component type.
    *
    * Note that the following `type`s add type-specific icons by default: `"date"`, `"email"`, `"password"`, `"search"`, `"tel"`, `"time"`.
+   *
+   *  The `"textarea"` type is [Deprecated], use the `calcite-text-area` component instead.
```

Comment:
Maybe for consistency with other value deprecations, the deprecation should be mentioned before the value? Perhaps something like:

```suggestion
   *  `"textarea"` [Deprecated], use the `calcite-text-area` component instead.
```